client/fullstack-book-manager-project/src/components/oneBookCard/OneBookCard.jsx

```diff
@@ -12,26 +12,25 @@ import { useDispatch } from "react-redux";
 import { setBooks } from "../../redux/books";
 
 const OneBookCard = ({
-  key,
-  cover,
-  title,
-  year,
-  author,
-  pages,
-  isFinished,
-  bookId,
-  userId,
-  currentUserId,
   children,
+  book,
 }) => {
   const dispatch = useDispatch();
-  const navigate = useNavigate();
+  
+  const { 
+    title,
+    year,
+    author,
+    pages,
+    isFinished,
+    _id:bookId,
```

Comment:
Destructuring rename is powerful way to change variable names whist destructuring